test/int/adapter.test.ts

```diff
@@ -15,12 +15,18 @@ const DATA_ROOT = testSetup.DATA_ROOT;
 
 suite('Chrome Debug Adapter etc', () => {
     let dc: ExtendedDebugClient;
+    let server;
+
     setup(() => {
         return testSetup.setup()
             .then(_dc => dc = _dc);
     });
 
     teardown(() => {
+        if (server) {
+            server.close();
```

Comment:
Should we assign null to server in case another tests starts and fails before assigning the server variable to a new server?

Reply:
Fair point. I changed all other instances of teardown to follow the same pattern.